packages/template-1c/package.json

```diff
@@ -12,16 +15,17 @@
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-helmet": "^6.1.0",
-        "react-router-dom": "^5.1.2"
+        "react-router-dom": "^5.1.2",
+        "typescript": "^4.5.5"
     },
     "scripts": {
         "format": "mrt format '\"**/*.{js,jsx}\"'",
-        "lint": "mrt lint '\"**/*.{js,jsx}\"'",
+        "lint": "tsc --noEmit && mrt lint '\"**/*.{js,jsx}\"'",
         "lint:fix": "npm run lint -- --fix",
         "save-credentials": "mrt save-credentials",
         "start": "mrt start",
         "test": "mrt test",
-        "build": "mrt build",
+        "build": "tsc --declaration --emitDeclarationOnly && mrt build",
```

Comment:
From our meeting earlier: maybe better to run `tsc` from within the `mrt build`.

Reply:
And for a project template like this template-1c, it probably doesn't make sense to generate declaration files. But it makes more sense for the sdk package.